python_modules/dagster/dagster/core/execution/context/input.py

```diff
@@ -297,6 +297,7 @@ def build_input_context(
     resource_config: Optional[Dict[str, Any]] = None,
     resources: Optional[Dict[str, Any]] = None,
     op_def: Optional[OpDefinition] = None,
+    step_context: Optional["StepExecutionContext"] = None,
```

Comment:
what's this about?  This doesn't seem to be used anywhere?

should we add to the doc section?

Reply:
thanks for catching - relic of an earlier version

Reply:
oops nvm - actually is used in assets_job.py. I'll add to the doc section